lib/fmt/token_fmt.ml

```diff
@@ -62,7 +62,8 @@ let string_of_token (t : token) : string  =
     | MINUSEQ -> "MINUSEQ"
     | TIMESEQ -> "TIMESEQ"
     | DIVEQ -> "DIVEQ"
-    | MODEQ -> "MODEQ"       
+    | MODEQ -> "MODEQ"
+    | QUESTION -> "TERNARY_IF_STATEMENT"       
```

Comment:
I suggest we format the token exactly as it's class name("QUESTION"), so that we could automatically generate this part in the future. 

and I believe ternary if statement is the only place `QUESTION` token will be used, so everyone knows what question means.

but it's a minor update, depends on you ;)

Reply:
yea true, I just wanted to be clear lol